mac

```diff
@@ -3,6 +3,16 @@
 # Be prepared to turn your laptop (or desktop, no haters here)
 # into an awesome development machine.
 
+SKIP_ZSH=0
+SHELL_CHANGES=()
+while getopts "z" opt; do
```

Comment:
This seems a little counter-intuitive to me, option '-z' means do _not_ install zsh.
Maybe we could have a more verbose flag like '--no-zsh' instead?

Reply:
I was feeling lazy about having to parse long options manually. =)

Could I use `-Z` to mean "skip zsh", and introduce `-z` as a no-op flag?

Reply:
Done!